ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -20,41 +20,131 @@ function getVersion() {
   echo "${version}"
 }
 
+function join_by { local IFS="$1"; shift; echo "$*"; }
+
+# BootstrapInstanceType gets its value from pkg/types/azure/defaults/platform.go
+architecture=${OCP_ARCH:-"amd64"}
+if [[ "${CLUSTER_TYPE}" == "azure-arm64" ]]; then
+  architecture="arm64"
+fi
+
 CONFIG="${SHARED_DIR}/install-config.yaml"
 
 REGION="${LEASED_RESOURCE}"
 echo "Azure region: ${REGION}"
 
+if [[ "${COMPUTE_NODE_TYPE}" == "" ]]; then
+  if [ "${architecture}" = "arm64" ]; then
+    COMPUTE_NODE_TYPE="Standard_D4ps_v5"
+  else
+    COMPUTE_NODE_TYPE="Standard_D4s_v3"
+  fi
+fi
 workers=3
 if [[ "${SIZE_VARIANT}" == "compact" ]]; then
   workers=0
 fi
 master_type=null
-if [[ "${SIZE_VARIANT}" == "xlarge" ]]; then
-  master_type=Standard_D32s_v3
-elif [[ "${SIZE_VARIANT}" == "large" ]]; then
-  master_type=Standard_D16s_v3
-elif [[ "${SIZE_VARIANT}" == "compact" ]]; then
-  master_type=Standard_D8s_v3
+if [ "${architecture}" = "arm64" ]; then
+  BOOTSTRAP_NODE_TYPE="Standard_D8ps_v5"
+  if [[ "${SIZE_VARIANT}" == "xlarge" ]]; then
```

Comment:
I think we can remove the double if.

We can:
- change the suffix (ps_v5 or s_v3) based on the arch.
- change the prefix (Standard_D(8|16|32)) based on the SIZE_VARIANT.

In this way, we don't repeat the if statement for the size variant two times.

Example:
```
if arch == arm64:
  BOOTSTRAP_NODE...
  master_type_suffix=ps_v5
else:
  BOOTSTRAP...
  master_type_suffix=s_v3

if SIZE_VARIANT...
  master_type_prefix=Standard_D32
.....

master_type=${master_type_prefix}${master_type_suffix}
```

Reply:
thanks for the suggests, will fix it